libcoap/src/types.rs

```diff
@@ -9,37 +9,37 @@
 
 //! Types required for conversion between libcoap C library abstractions and Rust types.
 
+use std::ffi::CString;
+use std::fmt::{Display, Formatter};
+use std::marker::PhantomPinned;
+use std::ops::{Deref, DerefMut};
+use std::pin::Pin;
 use std::{
     fmt::Debug,
     mem::MaybeUninit,
     net::{Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV6, ToSocketAddrs},
     os::raw::c_int,
     str::FromStr,
 };
-use std::ffi::CString;
-use std::fmt::{Display, Formatter};
-use std::marker::PhantomPinned;
-use std::ops::{Deref, DerefMut};
-use std::pin::Pin;
 
-use libc::{AF_INET, AF_INET6, c_ushort, in6_addr, in_addr, sa_family_t, sockaddr_in, sockaddr_in6, socklen_t};
```

Comment:
Some unrelated changes here – maybe the reformatting could also be moved into its own commit?

Reply:
I didn't make those changes intentionally, my IDE just randomly decided to "optimize" these imports.

I'll revert these changes, as they don't serve any real purpose.